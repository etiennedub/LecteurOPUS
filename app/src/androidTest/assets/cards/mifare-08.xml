<?xml version='1.0' encoding='UTF-8' standalone='yes' ?><Mifare><Pages><List index="0">043FAD1E</List><List index="1">12674980</List><List index="2">BC48E000</List><List index="3">00000003</List><List index="4">00000000</List><List index="5">3129C520</List><List index="6">6F5C0000</List><List index="7">6577CF63</List><List index="8">01C3078E</List><List index="9">78E01603</List><List index="10">FB0038D8</List><List index="11">2028EBA4</List><List index="12">01C3056E</List><List index="13">56E01004</List><List index="14">4800B8D8</List><List index="15">2028535E</List></Pages></Mifare>