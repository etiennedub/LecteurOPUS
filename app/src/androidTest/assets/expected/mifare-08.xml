<?xml version='1.0' encoding='UTF-8' standalone='yes' ?><Card><Type>OCCASIONNELLE</Type><ID>0</ID><Trip><List index="0" Date="04/10/2016" Time="16:07" Transfert="false" OperatorId="20" BusId="1" OperatorName="RTC" BusName="LEBUS 1" /><List index="1" Date="04/10/2016" Time="11:35" Transfert="false" OperatorId="20" BusId="1" OperatorName="RTC" BusName="LEBUS 1" /></Trip><Contract><List index="0" Subscription="false" Valid="true" NbTicket="0" OperatorId="20" OperatorName="RTC" /></Contract></Card>